### PR TITLE
gradle-profiler 0.17.0

### PR DESCRIPTION
**Pull request by anthonymonori**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
Hi. Thanks for your contribution.

The audit is complaining about the url:
```
2022-01-04T18:19:29.1014374Z [36mFormula/gradle-profiler.rb[0m:4:3: [33mC[0m: https://repo1.maven.org/maven2/org/gradle/profiler/gradle-profiler/0.17.0/gradle-profiler-0.17.0.zip should be [33mhttps://search.maven.org/remotecontent?filepath=org/gradle/profiler/gradle-profiler/0.17.0/gradle-profiler-0.17.0.zip[0m
2022-01-04T18:19:29.1016699Z   url "https://repo1.maven.org/maven2/org/gradle/profiler/gradle-profiler/0.17.0/gradle-profiler-0.17.0.zip"
2022-01-04T18:19:29.1017565Z   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2022-01-04T18:19:29.1019412Z [36mFormula/gradle-profiler.rb[0m:10:5: [33mC[0m: https://repo1.maven.org/maven2/org/gradle/profiler/gradle-profiler/maven-metadata.xml should be [33mhttps://search.maven.org/remotecontent?filepath=org/gradle/profiler/gradle-profiler/maven-metadata.xml[0m
2022-01-04T18:19:29.1021483Z     url "https://repo1.maven.org/maven2/org/gradle/profiler/gradle-profiler/maven-metadata.xml"
2022-01-04T18:19:29.1022592Z     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2022-01-04T18:19:29.1022853Z 
2022-01-04T18:19:29.1025837Z 5919 files inspected, [31m2 offenses[0m detected
```

**Comment by anthonymonori** [comment]
Yeah, noticed it once I opened the PR, although local audits all passed.

These new links, while different from the old one, it's what Gradle uses now: https://github.com/gradle/gradle-profiler/releases (see the "Prebuilt version available from" on the latest version).

Any idea how to proceed @iMichka ?

**Comment by iMichka** [comment]
This url audit was added here https://github.com/Homebrew/brew/pull/311
Some reasons as of why we try to use this URL can be found here: https://github.com/Homebrew/homebrew-core/pull/1610#discussion_r65311925
Some of these reasons are probably not correct anymore, nevertheless we try to be consistent and use the same urls everywhere. I checked and the suggested urls work correctly.

**Comment by anthonymonori** [comment]
@iMichka thanks for the links, that explains what needed to be changed. 

Updated the links according to the guidelines linked above.

File: Formula/gradle-profiler.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 class GradleProfiler < Formula
   desc "Profiling and benchmarking tool for Gradle builds"
   homepage "https://github.com/gradle/gradle-profiler/"
-  url "https://repo.gradle.org/gradle/ext-releases-local/org/gradle/profiler/gradle-profiler/0.16.0/gradle-profiler-0.16.0.zip"
-  sha256 "f376581ed7b788d9d3d640a2ddde88747ce2e8a0e297991a77b98e6b7a257fbb"
+  url "https://search.maven.org/remotecontent?filepath=org/gradle/profiler/gradle-profiler/0.17.0/gradle-profiler-0.17.0.zip"
+  sha256 "c71812c0fc79e23d8c9eafc831ba2f016bb7744935912b0aac1a98972dc30abe"
   license "Apache-2.0"
   revision 1
```
Comment (iMichka) [inline Formula/gradle-profiler.rb:7]: You should also remove the revision.

Comment (anthonymonori) [reply Formula/gradle-profiler.rb]: Fixed in c68dec4, thanks for pointing it out!

**Review by iMichka — APPROVED** [review]

**Comment by iMichka** [comment]
Thanks for your first contribution here @anthonymonori  !

**Pull request closed without merging**